<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>370</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnNext">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>370</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Suivant</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Veuillez entrer votre fichier d'en-tête (*.h):</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFilePath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFindPath">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>60</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom de la fonction</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>381</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Paramètres de la fonction</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtParams">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>381</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valeur de retour de la fonction</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtReturn">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>371</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Remarques sur la fonction</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtDetails">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>381</width>
      <height>84</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblSuivi">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>Fichier non chargé</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblHeader">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>361</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>En-tête</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Description de la fonction</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txtDesc">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>170</y>
      <width>371</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSkip">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Passer</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
